### ISSUE TITLE
TCGA Mutect2 and Strelka2 file appear to be identical? 

### ISSUE THREAD

**Issue by cansavvy** [AlexsLemonade/OpenPBTA-analysis#483]
#### What data file(s) does this issue pertain to?
`pbta-tcga-snv-mutect2.vep.maf.gz` and `pbta-tcga-snv-strelka2.vep.maf.gz`

#### What release are you using?
release-v13-20200116

#### Put your question or report your issue here.
Both have exactly 38296 number of mutation calls (number of rows) and the following prints out `TRUE`:
```
strelka_df <- data.table::fread("data/pbta-tcga-snv-strelka2.vep.maf.gz", data.table = FALSE)
mutect_df <- data.table::fread("data/pbta-tcga-snv-mutect2.vep.maf.gz", data.table = FALSE)

# This returns TRUE
all.equal(strelka_df, mutect_df)

# This returns TRUE as well
dplyr::all_equal(strelka_df, mutect_df)
```
Is there something I'm missing? Or did these file accidentally get mixed up?

**Comment by cansavvy** [comment]
I deleted the files and their redirects and re-ran `bash download-data.sh` and retried the above and am still getting `TRUE`? But the `md5sum.txt` file does show they are different.
 ```
93a91212e90b8c2533e8dfef9e72f7c2  pbta-tcga-snv-mutect2.vep.maf.gz
3c1f3fce62e4224fbb5466bd65516171  pbta-tcga-snv-strelka2.vep.maf.gz
```

**Comment by cansavvy** [comment]
In Terminal, `diff pbta-tcga-snv-mutect2.vep.maf.gz pbta-tcga-snv-strelka2.vep.maf.gz` doesn't spit back anything, but as a positive control, I tried  `diff pbta-tcga-snv-lancet.vep.maf.gz pbta-tcga-snv-strelka2.vep.maf.gz` and it showed me all the lines that were different.

**Comment by yuankunzhu** [comment]
@cansavvy i think we might have merged the same content for those two files. @tkoganti is checking and re-do it now. But a quick ls looks like those two files has a 1-byte file size off? 
```
2020-01-16 18:31:25    8805609 data/release-v13-20200116/pbta-tcga-snv-mutect2.vep.maf.gz
2020-01-16 18:31:26    8805610 data/release-v13-20200116/pbta-tcga-snv-strelka2.vep.maf.gz
```
that would explain the md5 difference, but with exactly same mutation content. sorry for the confusion, we will fix this in a new release.

**Comment by tkoganti** [comment]
Hi @cansavvy I used strelka files for mutect by mistake. Sorry about that. I uploaded new  strelka and mutect2 under V14 folder here - 
https://cavatica.sbgenomics.com/u/cavatica/pbta/files/#q?path=processed-data-merge%2FV14-data&search=pbta-tcga

mutect2 file is 3.3Mb and strelka is 8.4Mb (file size for new maf.gz files )

**Comment by cansavvy** [comment]
@tkoganti , this seems to be telling me I don't have access to the link you posted?

**Comment by cansavvy** [comment]
Also, the files before were 8.8Mb, but now neither file is that size?

**Comment by tkoganti** [comment]
Both the new and old files have a file size of 8.4 Mb (8,805,610 bytes) 

I did not realize you did not have access to cavatica. These files are in a staging area currently and we will let you know when they are released.

**Comment by jharenza** [comment]
@cansavvy if you have a CAVATICA username, we can add you to the project so you can preview the files. @tkoganti I am seeing that the Mutect2 gzipped maf is 3.3 MB and the Strelka2 gzipped maf is 8.4 MB - is this correct, or should Mutect2 be larger, as alluded to above?

**Comment by tkoganti** [comment]
@jharenza - See below for file sizes - 

new and old strelka gzipped maf files - ~8.4M
old mutect file - ~8.4M (This was an error)
New mutect file - ~3.3M

**Comment by jharenza** [comment]
closed via #507
